src/vega.schema.ts

```diff
@@ -689,6 +689,15 @@ const VG_MARK_CONFIG_INDEX: Flag<keyof BaseMarkConfig> = {
 
 export const VG_MARK_CONFIGS = keys(VG_MARK_CONFIG_INDEX);
 
+// Vega's cornerRadius channels.
+export const VG_CORNERRADIUS_CHANNELS = keys({
+  cornerRadius: 1,
```

Comment:
Why is this an object? We usually only use the set pattern (using 1 as value) with `Flag` since it lets us typecheck against keys of objects.

Reply:
If it' an array, the type will be `string[]`, which causes type check issues later in `getMarkConfig`.

Reply:
You can say `export const VG_CORNERRADIUS_CHANNELS = ['foo', 'bar'] as const`.